beets/importer.py

```diff
@@ -943,7 +943,9 @@ def is_archive(cls, path):
         if not os.path.isfile(path):
             return False
 
-        for path_test, _ in cls.handlers():
+        for path_test, x in cls.handlers():
+            if isinstance(path, bytes):
+                path = util.py3_path(path)
```

Comment:
and now I can't replicate it anymore, I must have accidentally fixed it while doing other bytestring stuff (unless it comes back to bite me later)